pulsar-broker/src/test/java/org/apache/pulsar/broker/service/PersistentMessageFinderTest.java

```diff
@@ -115,7 +115,7 @@ void testPersistentMessageFinder() throws Exception {
         config.setMaxEntriesPerLedger(entriesPerLedger);
         config.setRetentionTime(1, TimeUnit.HOURS);
         ManagedLedger ledger = factory.open(ledgerAndCursorName, config);
-        ManagedCursorImpl c1 = (ManagedCursorImpl) ledger.openCursor(ledgerAndCursorName);
```

Comment:
Seems the implementation of `openCursor(ledgerAndCursorName)` will default call `openCursor(ledgerAndCursorName, true)`, why do we need this change .

Reply:
In previous version code, we initialize the cursor at latest position, so the method `openCursor(ledgerAndCursorName)` has the same behavior with current method `openCursor(ledgerAndCursorName, true)`. Therefore, the new method `openCursor(ledgerAndCursorName )` is a compatible version of former api.
--